R/pkgapi.R

```diff
@@ -109,6 +110,14 @@ pkgapi_serialize_pass <- function(val, req, res, error_handler) {
 
 pkgapi_do_serialize_pass <- function(val, res) {
   res$setHeader("Content-Type", val$content_type)
+  if (!is.null(val$headers)) {
+    for (header in names(val$headers)) {
+      ## TODO: What do we want to do if a header already exists?
```

Comment:
Any thoughts on this would be good. Do we overwrite if it already exists or throw an error? I think this should only happen with `Content-Type` unless someone puts 2 headers with same name in the target function. That suggests we should error to me.

Reply:
Error sounds good